apps/src/templates/MultipleSectionsAssigner.jsx

```diff
@@ -229,6 +246,28 @@ MultipleSectionsAssigner.propTypes = {
 };
 
 const styles = {
```

Comment:
From what I know - we've agreed to use scss modules instead of inline styles. Why not using scss module here?

Reply:
Agreed –– I'm modifying MultipleSectionsAssigner as little as possible, as my team doesn't have capacity to refactor this component right now. But these styles should be in an SCSS module with updated styling. I believe Dani said you will help us with this?

Reply:
I don't know about it yet. Working on design system components currently and I thought that's the only thing I'm gonna help with for some time (a lot of work there)

Reply:
Okay, got it. I'll loop back with Dani after Codechella to see what the plan is!